### ISSUE TITLE
Python3 matplotlib 2.0.2 with Times New Roman misses unicode minus sign in pdf

### ISSUE THREAD

**Issue by gaomy3832** [matplotlib/matplotlib#9139]
### Bug report

With matplotlib 2.0.2 with python3.4.3, and "Times New Roman" font, saving figure as pdf file misses the unicode minus sign in the tick labels.

If I use pyplot.show() to directly show the figure, the figure is correct.

Also, only this particular combination of matplotlib and python version has this issue. I tried:

- python2.7.6, matplotlib 2.0.2
- python3.4.3, matplotlib 1.5.3

And they both work fine. Therefore I think it is not likely an issue with my environment, but rather a bug. But maybe I am missing something?

**Code for reproduction**

```python
import matplotlib
from matplotlib import pyplot as plt

matplotlib.rcParams['font.family'] = 'Times New Roman'

plt.plot([-1, 0, 1], [-1, 0, 1])

# plt.show()
plt.savefig('draw.pdf')
```

**Actual outcome**

When I open the file using Adobe Acrobat Reader DC, it says

```
An error exists on this page. Acrobat may not display the page correctly. Please contact the person who created the PDF document to correct the problem.
```

![image](https://user-images.githubusercontent.com/3386287/29935982-70c8b3f2-8e35-11e7-8024-6641745d028d.png)

**Matplotlib version**

  * Operating System: Ubuntu 14.04.4
  * Matplotlib Version: 2.0.2
  * Python Version: 3.4.3
  * Jupyter Version (if applicable):
  * Other Libraries: 

Executed in virtualenv. pip freeze output

```
cycler==0.10.0
matplotlib==2.0.2
numpy==1.13.1
pyparsing==2.2.0
python-dateutil==2.6.1
pytz==2017.2
six==1.10.0
```

If related, the LANG env var is set to "C". I also tried "en_US.UTF-8" and it didn't help.

**Comment by tacaswell** [comment]
I wonder if this will be fixed by https://github.com/matplotlib/matplotlib/pull/9059 and this is an issue embedding the fonts?

**Comment by gaomy3832** [comment]
Thanks for the reply.

I have checked out and tried the current master head 751c9af10eb844be08c6f841e67436c4662a960f. It does fix the minus sign issue, however the labels are now mistakenly in bold. I attach the two pdf files below. draw2.pdf is from python2 and draw3.pdf is from python3, with exactly the same code as included before. Is this an expected result?

[draw2.pdf](https://github.com/matplotlib/matplotlib/files/1269057/draw2.pdf)
[draw3.pdf](https://github.com/matplotlib/matplotlib/files/1269056/draw3.pdf)

**Comment by tacaswell** [comment]
No, they should be the same

**Comment by jsalort** [comment]
I have come into a problem which might be related to this issue.
I seem to have a problem generating PDF files with Times LaTeX lowercase greek letters in labels. The plot shows fine on screen but the generated PDF is broken. If you think this is unrelated to the present issue, please tell me so that I open another issue.
```python
from matplotlib import RcParams
import matplotlib.pyplot as plt

latex_style_times = RcParams({'font.family': 'serif',
               'font.serif': ['Times'],
               'text.usetex': True,
               })

plt.style.use(latex_style_times)
plt.figure()
plt.plot([1, 2, 3], [1, 2, 3], 'o')
plt.xlabel(r'$\delta\theta$')
plt.ylabel(r'$\Delta\Theta$')
plt.savefig('Essai.pdf')

plt.show()
```
[Essai.pdf](https://github.com/matplotlib/matplotlib/files/1622327/Essai.pdf)

This also yields an error message in Adobe Acrobat Reader DC:
> Unable to extract incorporated font StandardSymL_Slant_167.

**Comment by jkseppan** [comment]
@jsalort:
> I have come into a problem which might be related to this issue.
> I seem to have a problem generating PDF files with Times LaTeX lowercase greek letters in labels. The plot shows fine on screen but the generated PDF is broken. If you think this is unrelated to the present issue, please tell me so that I open another issue.
> [...]
> > Unable to extract incorporated font StandardSymL_Slant_167.

This is likely the same as #16087 which has a proposed fix in #18080.

**Comment by jkseppan** [comment]
The minus sign issue was probably fixed by #9059 and the boldface issue is something separate. I cannot reproduce it with current master, so please open a new issue if it is still happening.

**Comment by Qu246** [comment]
Hi
I am expecting a problem. 
Using Times New Roman font family, I cannot use \prime symbol anymore in my labels. 
I have understand that soe font family have issue with symbols susch as U+2032
But how can I deal with this using Times New Roman ? 
Thanks
